operate/client/src/App/DecisionInstance/Header/index.tsx

```diff
@@ -52,25 +50,28 @@ const getHeaderColumns = (isMultiTenancyEnabled: boolean = false) => {
   ];
 };
 
-const Header: React.FC = observer(() => {
-  const {
-    state: {status, decisionInstance},
-  } = decisionInstanceDetailsStore;
-  const {decisionInstanceId} = useParams<{decisionInstanceId: string}>();
+interface HeaderProps {
+  decisionEvaluationInstanceKey: string;
+}
+
+const Header: React.FC<HeaderProps> = ({decisionEvaluationInstanceKey}) => {
   const isMultiTenancyEnabled = window.clientConfig?.multiTenancyEnabled;
   const headerColumns = getHeaderColumns(isMultiTenancyEnabled);
   const tenantsById = useAvailableTenants();
+  const {data: decisionInstance, status} = useDecisionInstance(
+    decisionEvaluationInstanceKey,
+  );
 
-  if (status === 'initial') {
+  if (status === 'pending') {
```

Comment:
How do we want to handle users closing and opening a decision instance again?

React Query will serve the stale cache entry while refreshing the data in the background. The query `status` will still be `success`, since data is already available and can be shown.

Are we okay with showing cached data that likely will not change after the refresh, or do act like we have nothing cached, i.e. show the skeleton while refreshing?

Reply:
We need to confirm this but I believe the data of a decision instance never changes. So we could cache it for longer even

In general it's ok to show cached data until we fetch an update. We only show skeletons on the first load (even though this is not very consistent on the UI)